libpod/image/image.go

```diff
@@ -757,6 +757,20 @@ func (i *Image) Labels(ctx context.Context) (map[string]string, error) {
 	return imgInspect.Labels, nil
 }
 
+// GetLabel Returns a case-insensitive match of a given label
+func (i *Image) GetLabel(ctx context.Context, label string) (string, error) {
+	imageLabels, err := i.Labels(ctx)
+	if err != nil {
+		return "", err
+	}
+	labels := make(map[string]string)
```

Comment:
Do we need the map here? `strings.ToLower()` and then iterate through the labels, ToLower the key, and compare. Saves a bit of memory